app/i18n/locales/en.js

```diff
@@ -199,6 +199,7 @@ export default {
 	Message_pinned: 'Message pinned',
 	Message_removed: 'Message removed',
 	Messages: 'Messages',
+	Message_Reported: 'Message reported',
```

Comment:
you should also add translations for other languages using google translate :)

Reply:
In my personal experience, no translation is better than bad translation, someone familiar with the language should do it

Reply:
you can also look up here for translations https://github.com/RocketChat/Rocket.Chat.iOS/tree/develop/Rocket.Chat/Resources/de.lproj

Reply:
and as there are only a few words to be translated - google translate does this pretty well. and I agree that with large text strings google translate might make small mistakes

Reply:
IMO no translation is better than bad translation as well.

Reply:
`en` is fine